editor-footnotes.php

```diff
@@ -52,3 +52,18 @@ function autoload( $cls ) {
 
 // Load the Editor class on admin pages
 add_action( 'admin_init', [ '\Editor_Footnotes\Editor', 'instance' ] );
+
+/**
+ * Allow span[data-footnote] to pass through post content.
+ *
+ * @param  array $kses Allowed HTML tags.
+ * @param  string $context Kses context.
+ * @return array
+ */
+function wp_kses_allowed_html( $kses, $context ) {
+	if ( 'post' === $context ) {
```

Comment:
I guess it might be worth also double-checking that `$kses['span']` is there, in case someone tries to use this plugin and some other plugin or theme is disallowing them for some reason? Pretty unlikely but still...